nflow-engine/src/main/java/io/nflow/engine/internal/workflow/WorkflowDefinitionScanner.java

```diff
@@ -91,6 +94,15 @@
     return methods;
   }
 
+  public Set<WorkflowState> getStaticWorkflowStates(Class<?> definition) {
+    final Set<WorkflowState> states = new HashSet<>();
+    doWithFields(definition, field -> {
+      field.setAccessible(true);
```

Comment:
I would add try-catch around the set-accessible, or alternatively only run it if the field is not already accessible.
The newer javas are more strict on things.

Reply:
Added try-catch